components/popover/popover-mixin.js

```diff
@@ -0,0 +1,99 @@
+import '../colors/colors.js';
+import { css, html } from 'lit';
+
+const isSupported = ('popover' in HTMLElement.prototype);
+
+// eslint-disable-next-line no-console
+console.log('Popover', isSupported);
+
+export const PopoverMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			/**
+			 * Whether the popover is open or not
+			 * @type {boolean}
+			 */
+			opened: { type: Boolean, reflect: true },
+			_useNativePopover: { type: Boolean, reflect: true, attribute: 'popover' }
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				--d2l-popover-default-background-color: #ffffff;
+				--d2l-popover-default-border-color: var(--d2l-color-mica);
+				--d2l-popover-default-border-radius: 0.3rem;
+				--d2l-popover-default-foreground-color: var(--d2l-color-ferrite);
+				--d2l-popover-default-shadow: 0 2px 12px 0 rgba(0, 0, 0, 0.15);
+				background-color: transparent;
```

Comment:
We need to override many of these properties because the browser applies them when `popover` is set. We want to internalize these styles because we have not only the content container to deal with, but eventually, also the pointer.